main.go

```diff
@@ -1054,7 +1046,7 @@ func tabPrintSortedMap(headers []string, m map[string]string) {
 func getGoPath() (string, error) {
 	gp := os.Getenv("GOPATH")
 	if gp == "" {
-		return "", fmt.Errorf("GOPATH not set")
+		return homedir.Expand("~/go")
```

Comment:
I don't think this is safe to assume. For my setup this points to my go installation (aka GOROOT) and not my GOPATH